targets/arduino-mkr1000.json

```
@@ -0,0 +1,6 @@
+{
+    "inherits": ["atsamd21g18a"],
+    "build-tags": ["sam", "atsamd21g18a", "arduino_mkr1000"],
```

This was recently changed in #1490.

```suggestion
    "build-tags": ["arduino_mkr1000"],
```